packages/cli/src/commands/setup/ui/tasks/setup-component-library.js

```diff
@@ -0,0 +1,203 @@
+import fs from 'fs'
+
+import { getPaths } from '../../../../lib'
+
+/**
+ * @param {string} str The value to search for.
+ * @returns true if the contents of App.[js|tsx] contains the given string, false otherwise.
+ */
+export function appJSContains(str) {
+  const webAppPath = getPaths().web.app
+  const content = fs.readFileSync(webAppPath).toString()
+  return content.includes(str)
+}
+
+/**
+ * Inject code into App.[js|tsx].
```

Comment:
```suggestion
 * Inject code into App.{js,tsx}.
```

We usually use this "glob pattern" when talking about filenames